codecov.yml

```diff
@@ -0,0 +1,5 @@
+fixes:
```

Comment:
Not used can be removed

Reply:
I will remove

Reply:
Removed. Commit id is 3b2deca48bbe494c99803a2904afef9d2c7ed13c